### PR TITLE
[#29]: Add service worker implementation

### PR DESCRIPTION
**Pull request by sb7896**
This PR addresses #29
- Adding service-worker.js file
- Registered service-worker in index.html

### CODE REVIEW

File: client/service-worker.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+var cacheName = "notes-cache-v1";
```
Comment (skyshader) [inline client/service-worker.js]: Can we not move this into public directory?

Comment (aadityataparia) [reply client/service-worker.js]: Why not?

**Comment by aadityataparia** [comment]
Hi I would Like to work on this, to make a more detailed sw, what to cache, when to cache, where to cache, etc. Can I help?

**Comment by skyshader** [comment]
So, just to be clear, we already have a service worker setup in out client project. Do check that out first. And let's improve on top of that. IMO, this PR is doing what is already done.

@aadityataparia please submit a new PR if you want to improve upon the existing implementation, we would appreciate it :)

I am closing this for now.

**Pull request closed without merging**